src/Workspaces/Remote/Core/Services/RoslynServices.cs

```diff
@@ -32,13 +32,9 @@ internal sealed class RoslynServices
         // TODO: probably need to split this to private and public services
         public static readonly ImmutableArray<Assembly> RemoteHostAssemblies =
             MefHostServices.DefaultAssemblies
-                // This adds the exported MEF services from Workspaces.Desktop
-                .Add(typeof(Host.TemporaryStorageServiceFactory.TemporaryStorageService).Assembly)
                 // This adds the exported MEF services from the RemoteWorkspaces assembly.
                 .Add(typeof(RoslynServices).Assembly)
-                .Add(typeof(ICodingConventionsManager).Assembly)
-                .Add(typeof(CSharp.CodeLens.CSharpCodeLensDisplayInfoService).Assembly)
-                .Add(typeof(VisualBasic.CodeLens.VisualBasicDisplayInfoService).Assembly);
+                .Add(typeof(ICodingConventionsManager).Assembly);
```

Comment:
nice!

Reply:
This was actually a bug (duplicate of the features assemblies). Looks like we never had any exports in them before.